.github/workflows/reusable-lint.yaml

```diff
@@ -153,14 +156,24 @@ jobs:
             exit 1
           fi
       - name: Install flake8
-        if: needs.get-job-flags.outputs.CI_LINT_FLAKE8 == 'true'
+        # Always run flake8 if requested, whether other jobs succeeded or failed.
+        # Use ! cancelled so it will NOT run if the job was manually cancelled.
+        if: ${{ ! cancelled() && needs.get-job-flags.outputs.CI_LINT_FLAKE8 == 'true' }}
```

Comment:
Just curious, what is the effect of `! cancelled()` here?

Reply:
It runs in all cases unless the job was manually cancelled.

**        # Use ! cancelled so it will NOT run if the job was manually cancelled.**

Makes sure every linting job runs every time, so we can see exactly what failed, and it doesn't just stop running after the first failure (except if you manually cancel it, then it kills them all).

Reply:
Ah, okay, that makes sense! Thanks!